signac/contrib/job.py

```diff
@@ -90,7 +90,7 @@ def _save(self):
         # All elements of the job list are shallow copies of each other, so any
         # one of them is representative.
         job = next(iter(self._jobs))
-        old_id = job._id
+        old_id = job.id_
```

Comment:
We may need to revert some of the changes in this file, like `job.id_` back to `job._id` and `self.id_` back to `self._id`. That is meant to be a private attribute because of the leading `_`. Also attribute names do not shadow the builtin `id`. cc: @vyasr for thoughts on this.

Reply:
Agreed, in this case the leading underscore is correctly indicating the intent of making a variable internal ("private")